tests/installation/zdup.pm

```diff
@@ -150,7 +151,8 @@ sub run() {
         elsif ($out =~ $zypper_retrieving or $out =~ $zypper_check_conflicts) {
             # probably to avoid hitting black screen on video
             send_key 'shift', 1;
-            # continue but do not drop zypper_dup_fileconflict check
+            # continue but do a check again
+            $out = wait_serial($post_checks, 240);
```

Comment:
I may be distracted or what, but it actually took me almost a minute to read the comment properly. I always read `# continue but do NOT check again` :D